packages/datadog-plugin-graphql/test/index.spec.js

```diff
@@ -833,6 +853,18 @@ describe('Plugin', () => {
           graphql.execute({ schema, document })
         })
 
+        it('should not re-execute thenables from resolvers', done => {
+          const source = '{ human { oneTime } }'
+
+          graphql.graphql({ schema, source })
+            .then(result => {
+              expect(result).to.not.have.property('errors')
+              expect(result.data.human.oneTime).to.equal('one-time result')
+              done()
+            })
+            .catch(done)
+        })
```

Comment:
```suggestion
        it('should not re-execute thenables from resolvers', async () => {
          const source = '{ human { oneTime } }'

          const result = await graphql.graphql({ schema, source })
          expect(result).to.not.have.property('errors')
          expect(result.data.human.oneTime).to.equal('one-time result')
        })
```

I guess that should also be fine to use?

Reply:
The case for this was a Mongoose V6 query object that is only expected to be called once. Here's the docs explaining it: https://mongoosejs.com/docs/migrating_to_6.html#duplicate-query-execution.

Reply:
I've updated the tests to use await syntax.